src/me/mrCookieSlime/Slimefun/Setup/SlimefunSetup.java

```diff
@@ -1831,10 +1832,48 @@ public boolean onBlockBreak(BlockBreakEvent e, ItemStack item, int fortune, List
 				else return false;
 			}
 		});
+                
+		new SlimefunMachine(Categories.MACHINES_1, SlimefunItems.TABLE_SAW, "TABLE_SAW",
+		new ItemStack[] {null, null, null, new ItemStack(Material.BIRCH_SLAB), new ItemStack(Material.STONECUTTER), new ItemStack(Material.BIRCH_SLAB), null, new ItemStack(Material.IRON_BLOCK), null},
```

Comment:
I think you should rather use Stone Slabs instead of Birch Slabs.
Birch is a very specific type of wood.